Polyhedron/demo/Polyhedron/Plugins/Surface_mesh/Offset_meshing_plugin.cpp

```diff
@@ -507,9 +509,10 @@ void Polyhedron_demo_offset_meshing_plugin::offset_meshing()
                                tag_index);
   connect(worker, &QThread::finished, worker, &QObject::deleteLater);
   connect(worker, &Mesher_thread::resultReady, this, 
-          [item, angle, sizing, approx, offset_value, index]
+          [this, item, angle, sizing, approx, offset_value, index]
           (SMesh *new_mesh){
     QApplication::restoreOverrideCursor();
+    static_cast<MainWindow*>(mw)->lock_test_item(false);
```

Comment:
That call to `lock_test_item` should be at the end of the lambda body. That does not change anything, because this lambda will be interpreted by the main thread, but it does not "look right", if one thinks of `lock_test_item(true)/lock_test_item(false)` as the locking of a resource.